### DISCUSSION TITLE
Seeking Expertise: UDOIT Installation on RedHat (Docker) in Higher Ed Environment

### DISCUSSION THREAD

**Discussion by leturnoMVCC** [ucfcdl/UDOIT#920] [Q&A]
Hello everyone,

I am the Canvas administrator for Moraine Valley Community College, and our Unix administration team is currently facing challenges with a fresh installation of the latest version of UDOIT on a RedHat Linux VM using Docker.

We appear to be running into roadblocks installing the UDOIT docker install on Redhat.

We're eager to connect with an IT specialist from another higher education institution who has successfully navigated this installation. Our objective is to ensure a smooth setup and benefit from any best practices or insights you may have gained from your experience.

Here's a brief overview of our current scenario:

- Platform: RedHat Linux (latest version)
- Environment: VM
- Installation Method: Docker
- UDOIT Version: Latest Version

Specifically, we're looking for guidance on:

Guidance on handling challenges of the docker install on Redhat.
Any prerequisites or preparations needed before the installation.
Common pitfalls or challenges you encountered and how you addressed them.
Post-installation configurations or optimizations that helped stabilize the platform.
If you've been through a similar setup or can recommend someone who has, we'd greatly appreciate a few minutes of your time. Your insights will not only benefit our institution but also contribute to the broader academic community looking to leverage UDOIT in their learning environments.

Thank you in advance for your support and looking forward to connecting with the experts in this community!

Warm regards,

Scott Leturno
Canvas Administrator
Center for Teaching and Learning
Moraine Valley Community College

**Comment by ssilverm** [comment]
I'd be willing to help out.  My name is Shea Silverman, I used to work at UCF and helped create the docker container. 

You can contact me at silver2k AT gmail DOT com
